ee/allowedcmd/cmd_linux.go

```diff
@@ -16,7 +16,7 @@ func Apt(ctx context.Context, arg ...string) (*exec.Cmd, error) {
 func Brew(ctx context.Context, arg ...string) (*exec.Cmd, error) {
 	validatedCmd, err := validatedCommand(ctx, "/home/linuxbrew/.linuxbrew/bin/brew", arg...)
 	if err != nil {
-		return nil, err
+		return nil, nil
```

Comment:
I think this should still be `nil, err`, so we can return the wrapped `ErrCommandNotFound` to `upgradeable.go` to check